courtroom.cpp

```diff
@@ -85,6 +85,7 @@ void Courtroom::enter_courtroom(int p_cid)
 
   set_emote_page();
   set_emote_dropdown();
+  ui_pre->setChecked(ui_pre || ao_config->always_pre_enabled());
```

Comment:
This would fix https://github.com/Chrezm/DRO-Client/issues/51, no?

Reply:
It wouldn't fix `ui_pre` for the sound case yet.